src/haswell/intrinsics.h

```diff
@@ -4,7 +4,14 @@
 #include "simdjson.h"
 
 #ifdef _MSC_VER
-#include <intrin.h> // visual studio
+// visual studio (or clang-cl)
+#include <intrin.h>
+#ifdef __clang__
+#include <bmiintrin.h>
```

Comment:
Though, admitted, we probably crash and burn under ARM with Visual Studio, this is of course not going to work... e.g., if you try to use Visual Studio to compile for android with clang, this will blow up.